tests/requirements.txt

```diff
@@ -13,6 +13,6 @@ docker
 better_exceptions
 pyasn1
 Jinja2
-httpx == 0.21.*
+httpx
```

Comment:
I am reluctant to let go of this, because httpx does break between minor releases (which is their prerogative under semver). I would rather we pinned to 0.22.*